man/systemd.exec.xml

```diff
@@ -1187,7 +1187,76 @@
         <function>read</function> and <function>write</function>, and
         right after it add a blacklisting of
         <function>write</function>, then <function>write</function>
-        will be removed from the set.) </para></listitem>
+        will be removed from the set.)</para>
+
+        <para>As the number of possible system
+        calls is large, predefined sets of system calls are provided.
+        A set starts with <literal>@</literal> character, followed by
+        name of the set.
+
+        <table>
+          <title>Currently predefined system call sets</title>
+
+          <tgroup cols='2'>
+            <colspec colname='set' />
+            <colspec colname='description' />
+            <thead>
+              <row>
+                <entry>Set</entry>
+                <entry>Description</entry>
+              </row>
+            </thead>
+            <tbody>
+              <row>
+                <entry>@clock</entry>
+                <entry>System clock setting</entry>
+              </row>
+              <row>
+                <entry>@default</entry>
+                <entry>Above mentioned default list</entry>
```

Comment:
Well, should we implement it? And if we should, should we also document it? I mean, this might be confusing. We quite often implement features or niche concepts in existing features that we deliberately don't document, in order to avoid confusion and keep something as an undocumented, and thus subject-to-removal feature...